Resources/Prototypes/Entities/Objects/Specific/Xenoarchaeology/artifacts.yml

```diff
@@ -27,6 +27,21 @@
     - type: Appearance
       visuals:
         - type: RandomArtifactVisualizer
+    - type: PowerConsumer
+      voltage: Medium
+      drawRate: 500
+    - type: NodeContainer
+      nodes:
+        medium:
+          !type:CableDeviceNode
+          nodeGroupID: MVPower
+          # sadly, HVPower and Apc cables doesn't work right now
```

Comment:
why not? is there a bug here

Reply:
Yeah, check #6724. HVPower and Apc cables are always has 0 power for some reason.

Reply:
weird, alright